Package.swift

```diff
@@ -203,6 +203,7 @@ var package = Package(
                 "Runtime",
                 "Swallow",
                 "SwallowMacrosClient",
+                .product(name: "SwiftSyntaxMacrosTestSupport", package: "swift-syntax", condition: .when(platforms: [.macOS])),
```

Comment:
@aksh1t remove this, add it to the test target.

Reply:
It is already in the test target. It looks confusing in the GitHub web UI because the code is folding in a way that you'd think it is in the main target and not the test target.

Reply:
@aksh1t my bad, you're right.